ospfd/ospf_zebra.c

```diff
@@ -1351,11 +1351,14 @@ static int ospf_distribute_list_update_timer(struct thread *thread)
 						default_refresh = 1;
 					else if (
 						(lsa = ospf_external_info_find_lsa(
-							 ospf, &ei->p)))
+							 ospf, &ei->p))) {
+						int force =
+							LSA_REFRESH_IF_CHANGED;
```

Comment:
Looks Good to me.  Small suggestion , adding a comment about this scenario here really helps.

Reply:
Done